src/Microsoft.Data.Analysis/DataFrame.cs

```diff
@@ -597,15 +597,45 @@ private void OnColumnsChanged()
         private DataFrame Sort(string columnName, bool isAscending)
         {
             DataFrameColumn column = Columns[columnName];
-            DataFrameColumn sortIndices = column.GetAscendingSortIndices();
+
+            List<long?> sortIndicesNonNull = column.GetAscendingSortIndices().ToList();
```

Comment:
Why `ToList()` here?

Reply:
The reasoning behind it was so that i could use the AddRange method further down. Which would allow me to append the omitted values (Nulls) to the front or end of the list.